ludwig/models/trainer.py

```diff
@@ -1039,12 +1041,13 @@ def train_online(self, dataset):
         ) as batcher:
 
             # training step loop
-            progress_bar = tqdm(
-                desc="Training online",
-                total=batcher.steps_per_epoch,
-                file=sys.stdout,
-                disable=is_progressbar_disabled(),
-            )
+            progress_bar_config = {
+                "desc": "Training online",
+                "total": batcher.steps_per_epoch,
+                "file": sys.stdout,
+                "disable": is_progressbar_disabled(),
+            }
+            progress_bar = LudwigProgressBar(self.report_tqdm_to_ray, progress_bar_config, True)
```

Comment:
Why is this constructed with `True` as the argument for `is_coordinator` directly instead of `self.is_coordinator()`?

Reply:
good find